metalearners/drlearner.py

```diff
@@ -331,6 +331,35 @@ def evaluate(
 
         return variant_outcome_evaluation | propensity_evaluation | treatment_evaluation
 
+    def treatment_effect(
+        self,
+        X: Matrix,
+        y: Vector,
+        w: Vector,
+    ) -> np.ndarray:
+        """Compute Average Treatment Effect (ATE) for each treatment variant using the Augmented IPW estimator (Robins et al 1994). Does not require fitting a second-stage treatment model: it uses the pseudo-outcome alone and computes the average and SE. Can be used following the `fit_all_nuisance` method.
```

Comment:
```suggestion
        """Compute Average Treatment Effect (ATE) for each treatment variant using the Augmented IPW estimator (Robins et al 1994). Does not require fitting a second-stage treatment model: it uses the pseudo-outcome alone and computes the average and SE. Can be used following the :meth:`~metalearners.drlearner.DRLearner.fit_all_nuisance` method.
```

This way a user can get a cross-reference in the docs. :)